code/__HELPERS/global_lists.dm

```diff
@@ -47,8 +47,8 @@ var/list/mannequins_
 var/global/list/all_species[0]
 var/global/list/all_languages[0]
 var/global/list/language_keys[0]					// Table of say codes for all languages
-var/global/list/whitelisted_species = list("Human") // Species that require a whitelist check.
-var/global/list/playable_species = list("Human")    // A list of ALL playable species, whitelisted, latejoin or otherwise.
+var/global/list/whitelisted_species = list("Human", "Kidan") // Species that require a whitelist check.
```

Comment:
this got a lot of "kidan" stuff. not to mention a lot of other code that got nothing to do with sprites.